security/keycloak-authz/src/main/java/io/quarkus/ts/openshift/security/keycloak/authz/UserResource.java

```diff
@@ -20,13 +21,15 @@
 
     @GET
     @Produces(MediaType.TEXT_PLAIN)
+    @Authenticated
```

Comment:
This annotation is no longer necessary. This was motivated by an issue that was fixed before 2.0.0.Final was out.